R/cre.R

```diff
@@ -109,6 +108,7 @@
 cre <- function(y, z, X, method_params, hyper_params){
 
   # Input checks ---------------------------------------------------------------
+  logger::log_info("Load dataset")
```

Comment:
If this is the beginning of the process, let's use present participle when the process starts with trailing ... 
For example, this should be: Loading dataset ...